indexing-service/src/main/java/org/apache/druid/indexing/worker/shuffle/IntermediaryDataManager.java

```diff
@@ -53,7 +56,7 @@
    *
    * @return size of the writen segment
```

Comment:
This method no longer returns the size written. Can you fix the javadoc as well?